src/main/resources/application.properties

```
@@ -1 +1,10 @@
 spring.application.name=mini-survey-monkey
+spring.datasource.url=jdbc:h2:file:/Users/ali/Documents/SYSC4806/mini-survey-monkey/src/main/resources/Data/SYSC4806;AUTO_SERVER=true
```

You mentioned this in your PR description, but this probably shouldn't be stored in the repo.

Keeping it on the local file system like below should work
```suggestion
spring.datasource.url=jdbc:h2:file:/data/mini-survey-monkey;AUTO_SERVER=true
```